modules/ROOT/partials/misc/minimum-version.adoc

```diff
@@ -0,0 +1 @@
+NOTE: This feature is only available for {productname} {minimumversion} and later.
```

Comment:
This isn't how this has been done in the past and having to specify the `minimumversion` all the time is messy... see https://github.com/tinymce/tinymce-docs/pull/2471#discussion_r918548089 on how this should be done for this specific case.

Reply:
We have ended up with every PR doing it separately. This was not well coordinated.

Reply:
I have added a new partial file in misc named requires_6_1v.adoc in this PR and https://github.com/tinymce/tinymce-docs/pull/2467, just as in the linked example.

Reply:
@MitchCTiny that's not what I said to do... I said it needs coordination, not duplication.